discord/gateway.py

```diff
@@ -129,9 +132,11 @@ def ack(self):
         if self.latency > 10:
             log.warning(self.behind_msg, self.latency)
 
+
 class VoiceKeepAliveHandler(KeepAliveHandler):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.recent_ack_latencies = []
```

Comment:
A `deque` is the proper data type here since it handles automatically removing for you at the C level and has O(1) pop_front behaviour.

It resides in `collections`.